can_tools/bootstrap_data/covid_categories.csv

```diff
@@ -49,3 +49,6 @@ test,unspecified_tests_negative
 test,unspecified_tests_positive
 test,unspecified_tests_total
 test,reported_positivity
+hospital,critical_staff_shortage_yes
+hospital,critical_staff_shortage_no
+hospital,critical_staff_shortage_noreport
```

Comment:
This is only applicable for the state timeseries data.

It's related to the coverage information, but I think it also tells us a bit more (which is why I chose to include this one and not the other). In addition to giving us an idea of the number of hospitals that actually report, it is telling us how many of them are currently experiencing (or are anticipating) a staffing shortage.

Not sure if we would make use of it, but it seemed at least potentially relevant and was low cost to include.

Reply:
Cool, I'm definitely down to get the critical_staff_shortage data ingested! 🎉  

I was just highlighting that the "_noreport" column is an aspect that we aren't capturing in the rest of this data ingestion (all of the hospital utilization / capacity numbers could be incomplete and we don't know how incomplete they are without capturing the _coverage columns).

Reply:
I see what you mean now -- Completely agree that the `_noreport` column is akin to the `_coverage` data (which I've chosen not to collect for now).

I'm going to leave it for now because it doesn't cost much to have it (and saves me 15 seconds it would take to delete the right data :sweat_smile: ). I'll pitch it more optimistically as, "this way we have a way to estimate how many hospitals are reporting", but if we wanted to do something serious with information on the `_coverage` then we would want to bring it in for each relevant variable.

Reply:
👍